__tests__/plugins/formatters.commerce.test.ts

```diff
@@ -142,3 +142,31 @@ loader.paths('f-variants-select-%N.html').forEach((path) => {
 loader.paths('f-variants-select-subscription.html').forEach((path) => {
```

Comment:
All of these tests and their corresponding html files were copied from [template-compiler](https://github.com/Squarespace/template-compiler/tree/main/core/src/test/resources/com/squarespace/template/plugins/platform)